build.gradle

```diff
@@ -23,6 +22,10 @@ ext.deps = [
   'jimfs'         : 'com.google.jimfs:jimfs:1.1'
 ]
 
+if (gradle.ext.isKotlinDev) {
+  project.properties.put("VERSION_NAME", "0.38.0-kotlin-dev-SNAPSHOT")
```

Comment:
Shouldn't the version number be dynamic?

Reply:
`-SNAPSHOT` means it will be dynamic

Reply:
@Tapchicoma agree here. Can we pull the main version name from the `gradle.properties` file or something? Otherwise we have to remember to update `0.38.0` in 2 places when we next release.

Reply:
sorry, missed here main point. Will update code to pull it from `gradle.properties`

Reply:
Fixed via https://github.com/pinterest/ktlint/pull/809/commits/8f4fa138bb1f2dd972b001d4fd1773f511c5afe0